airflow/models/abstractoperator.py

```diff
@@ -696,6 +696,9 @@ def get_template_env(self, dag: DAG | None = None) -> jinja2.Environment:
             dag = self.get_dag()
         return super().get_template_env(dag=dag)
 
+    def render_value(self, context: Context, jinja_env: jinja2.Environment, value):
+        return self.render_template(value, context, jinja_env)
```

Comment:
DO we still need `render_value` at this point…? Maybe we should just tell people to use `render_template`, which is already public.

Reply:
done